src/components/Form/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+	}
+
+	submitWrapper() {
+		if (typeof(this.props.onSubmit) !== "undefined") {
+			this.props.onSubmit();
+		}
+		alert('Form submitted successfully!');
```

Comment:
alert is kind of janky, remove for now and we can create a new UI component to convey this info later